bliss/models/galaxy_net.py

```diff
@@ -110,50 +136,68 @@ def generate_latents(self, dataloader, n_batches):
             for _ in tqdm(range(n_batches)):
                 galaxy = next(iter(dataloader))
                 noiseless = galaxy["noiseless"].to(self.device)
-                latent_batch = enc(noiseless, 0.0)
+                latent_batch, _ = enc(noiseless, 0.0)
```

Comment:
why use the noiseless version of the images here? Doesn't the encoder assume imatges have noise ?

Reply:
Good catch - this was left over from before. I fixed it to use the images and background, but I may just delete this method since we don't use it anymore.

Reply:
I actually went further and just deleted the method since we don't use it anymore.